logs/logs.go

```diff
@@ -19,6 +19,8 @@ func SetAllLoggers(level logging.LogLevel) {
 	_ = logging.SetLogLevel("basichost", "INFO")
 	_ = logging.SetLogLevel("pubsub", "WARN")
 	_ = logging.SetLogLevel("net/identify", "ERROR")
+	_ = logging.SetLogLevel("shrex/nd", "ERROR")
+	_ = logging.SetLogLevel("shrex/eds", "ERROR")
```

Comment:
Why?

Reply:
Explained in PR description. I want to hide warn logs from shrex client/server by default.

Reply:
Ok, the logic of which level means what makes total sense, but I still don't get why we want to hide warning logs from operators(and from us because they report what they see).  Also operators seems to follow the logic you described and won't panic on Warns

Reply:
Yeah, I agree. Lets make default WARN